packages/lib/src/utils/Transactions.js

```diff
@@ -79,6 +79,10 @@ export async function sendDataTransaction(contract, txParams) {
   return contract.sendTransaction({ gas, ... txParams });
 }
 
+export async function getAccount(index) {
+  return web3.eth.accounts[index];
```

Comment:
AFAIK this is not async, I think it would be clearer using a promisified version of `getAccounts`